tests/framework/di/ContainerTest.php

```diff
@@ -95,6 +97,19 @@ public function testDefault()
         $this->assertInstanceOf($Bar, $foo->bar);
         $this->assertInstanceOf($Qux, $foo->bar->qux);
 
+        // predefined property parameters
+        $fooSetter = FooProperty::class;
+        $barSetter = BarSetter::class;
```

Comment:
At least `travis-ci` is right about that: needs `::className()` for PHP 5.4 compability.

Reply:
Fixed